### ISSUE TITLE
'docker run ...' doesn't do anything from Linux subsystem on Windows

### ISSUE THREAD

**Issue by ceddlyburge** [exercism/elm-representer#3]
The 'docker run ...' command doesn't do anything when running linux subsystem (GitBash) on windows.

It does run the container, which you can see with `docker ps -a`, but using `docker logs ....` doesn't show any output, so presumably the container is created, does nothing, and then exits.

This isn't a huge deal, as you can just run it in powershell or the command line, but it does mean that comprehensive testing is not possible.

This is turn means that ./bin/run-in-docker.sh doesn't work properly on windows. It does create the docker image successfully, but again the `docker run` command doesn't do anything.

**Comment by mpizenberg** [comment]
@ceddlyburge is this still not working in Linux subsystem on Windows in PR #5 ?

**Comment by ceddlyburge** [comment]
Hi Matthieu. I checked out the update-docker branch

If I run this command in git bash:

` ./bin/run-in-docker.sh space-age ../space-age ../space-age-output`

I get the following error, which happens after building the image:

'cp: cannot stat 'C:/Users/cburge/AppData/Local/Programs/Git/solution/': No such file or directory'

The same thing happens if I use absolute paths.

I haven't tried to debug it, and it is a different problem than the earlier one, and the path is from my local computer, rather than the docker image, so it looks like something weird is going on. I'm not sure how much it matters though.

**Comment by mpizenberg** [comment]
Weird to see  a path ending with `solution/` in your error message. The only place where that is used in a path is when calling `docker run ... dst=/solution ...` and so this path is within the container so I don't know where that `C:/Users/......Git/` is coming from before the `solution` ...

**Comment by ceddlyburge** [comment]
Yeah, something strange. I'm not sure its worth digging in to though, as long as the bash script works on linux and the powershell script works on windows I think we're good (and presumably the bash script on linux is the only thing that is actually needed)
